cmd/controlplane/controller.go

```diff
@@ -32,305 +33,367 @@ import (
 	_ "github.com/akuity/kargo/internal/gitprovider/github"
 )
 
+type controllerOptions struct {
+	ShardName  string
+	KubeConfig string
+
+	ArgoCDEnabled       bool
+	ArgoCDKubeConfig    string
+	ArgoCDNamespaceOnly bool
+
+	RolloutsEnabled      bool
+	RolloutsKubeConfig   string
+	AnalysisRunNamespace string
```

Comment:
Both of these went away in #1706.

AnalysisRuns now live exclusively in the Kargo control plane, so there's never a need for a separate client nor a need to fall back on using any namespace that isn't a Project's own namespace.